integration_tests/saiutil/saiutil.go

```diff
@@ -103,3 +131,138 @@ func CreateNeighbor(t testing.TB, dut *ondatra.DUTDevice, ip string, dmac string
 		t.Fatal(err)
 	}
 }
+
+type InterfaceRef struct {
+	Intf    string
+	SubIntf uint32
+}
+
+func ConfigAft(vrf string, aft *oc.NetworkInstance_Afts) *configer {
+	c := &configer{}
+
+	c.cfg = func(t testing.TB, s *Suite, dut *ondatra.DUTDevice) {
+		conn := dataplaneConn(t, dut)
+		nhc := saipb.NewNextHopClient(conn)
+		nhgc := saipb.NewNextHopGroupClient(conn)
+
+		for id, nh := range aft.NextHop {
+			ref := InterfaceRef{
+				Intf:    nh.GetInterfaceRef().GetInterface(),
+				SubIntf: nh.GetInterfaceRef().GetSubinterface(),
+			}
+			req := &saipb.CreateNextHopRequest{
+				Switch:            1,
+				Type:              saipb.NextHopType_NEXT_HOP_TYPE_IP.Enum(),
+				RouterInterfaceId: proto.Uint64(s.interfaceMap[ref]),
+				Ip:                net.ParseIP(*nh.IpAddress),
+			}
+			resp, err := nhc.CreateNextHop(context.Background(), req)
+			if err != nil {
+				t.Fatalf("failed to create next hop %d: %v", id, err)
+			}
+			s.oc2SAINextHop[id] = resp.GetOid()
+		}
+
+		for id, nhg := range aft.NextHopGroup {
+			req := &saipb.CreateNextHopGroupRequest{
+				Type: saipb.NextHopGroupType_NEXT_HOP_GROUP_TYPE_ECMP_WITH_MEMBERS.Enum(),
+			}
+			for id, nh := range nhg.NextHop {
+				req.NextHopList = append(req.NextHopList, s.oc2SAINextHop[id])
+				req.NextHopMemberWeightList = append(req.NextHopMemberWeightList, uint32(nh.GetWeight()))
+			}
+			resp, err := nhgc.CreateNextHopGroup(context.Background(), req)
+			if err != nil {
+				t.Fatalf("failed to create next hop group %d: %v", id, err)
+			}
+			s.oc2SAINextHopGrp[id] = resp.GetOid()
+		}
+
+		for pre, entry := range aft.Ipv4Entry {
+			CreateRoute(t, dut, pre, s.oc2SAINextHopGrp[entry.GetNextHopGroup()], s.oc2SAIVRF[vrf])
+		}
+		for pre, entry := range aft.Ipv6Entry {
+			CreateRoute(t, dut, pre, s.oc2SAINextHopGrp[entry.GetNextHopGroup()], s.oc2SAIVRF[vrf])
+		}
+	}
+	c.uncfg = func(t testing.TB, s *Suite, dut *ondatra.DUTDevice) {}
+
+	return c
+}
+
+func ConfigRIF(portID string, mac string, vlan string) *configer {
```

Comment:
vlan is not used, is it a place holder?

Reply:
had some local changes i forgot to commit. this should be VRF